subsys/bluetooth/controller/hal/nrf5/radio/radio.c

```diff
@@ -665,16 +654,7 @@ u32_t radio_tmr_start(u8_t trx, u32_t ticks_start, u32_t remainder)
 	HAL_SW_SWITCH_TIMER_CLEAR_PPI_REGISTER_TASK =
 		HAL_SW_SWITCH_TIMER_CLEAR_PPI_TASK;
 
-#if defined(CONFIG_SOC_NRF52840)
```

Comment:
We use a single PPI for S2 and we don't (by default) assign it to a group, so we don't need different compile-time asignments for 832 and 840.